broker-daemon/models/fill.js

```diff
@@ -398,6 +399,41 @@ class Fill {
     return fill
   }
 
+  /**
+   * serialize an fill for transmission via grpc
+   * @param {Object} fillObject - Plain object representation of the fill, state, dates
+   * @returns {Object} Object to be serialized into a GRPC message
+   */
+  static serialize (fillObject) {
+    const {
+      fill,
+      state,
+      error,
+      dates
+    } = fillObject
+
+    const { order } = fill
+
+    const baseAmountFactor = CONFIG.currencies.find(({ symbol }) => symbol === order.baseSymbol).quantumsPerCommon
```

Comment:
Do we use the engine config for this in other places? I think we are trying to transition to using engine config where possible.

Reply:
Can you link an example of where we use engineConfig?

Reply:
https://github.com/sparkswap/broker/blob/master/broker-daemon/broker-rpc/wallet-service/commit.js#L51

Reply:
Maybe we should pass in this kind of config when we initialize the model rather than pulling it out during serialization?